op-tests/test-blspy-g1.txt

```diff
@@ -0,0 +1,120 @@
+bls_g1_multiply 0xa6f2b01ae6df217cfb69311792aa7a9739276b344026add94be0508a36159cbe3968995a20fc8ff9dd041f19e52b8ba0 -43 => 0xa3d050971d7d1461dcf8e0ea78871d5341cb132eb6da1321dbadd18929f5aa9d7eec8cc162419d248d4ee4cd2cabd77f | 2154839
+bls_g1_multiply 0x8c578db43629a04e5eb3277aeba89cd6101bf98dd1ae6ae2dddbbfd305fac0d263815063e3a040e23a4ecfebf439bc09 28 => 0x882ce98b7d8e12ad1feafcad8b63ccca8485cbb612734ce26b213a8c29ad7d5f77a74bd1a54441990f072853578f234e | 2154839
+bls_g1_multiply 0xa4a108e5dabd6d6fde629fe9515cf397ab2a2d44c5740d39525aa7f5230df38d6753cb7a0683a75e2f4f3d82fd46cdb0 -53 => 0xb0f7828d279d4f6dc257647c6e2cdd7483180668e3efcf5146ed54e0c3faceef822d3780f69bc424424a5fd8d4387af9 | 2154839
+bls_g1_multiply 0x89c84dff22dc5c15698fb9476b602cee2d28efc217ebf6218053cfea1885676bf4f6b93895ea4e6321928d49c226ecfc 11 => 0xb57a91f9b1473c307aef099cd81de290133e0bff6cb6cac9186b9f2c1deeb65abddfcb6f88d87e6884a41b8fb6e8c728 | 2154839
+bls_g1_multiply 0x8d84cd1c33f37c64b864f8c042b6a11298f42618b4dfafb7ce43bd5142bd302232d096baf05c6ea6dc090ed8576fc7ab -95 => 0x8996bbcf72c77c563478e7e3966c950090dd1f050f9d54306d73b5162a798ad28d182d71271a192af11990ccc69a5427 | 2154839
+bls_g1_multiply 0xa45ea6d6d4955092665062c8806ad271b1a2394d27897002f30a45089696146703bbbc6634f269821a2e82d1e4d8e509 8 => 0xa0b1aa4d59f997efd03adca9b0aab10a445ee2b2b3b2afbaae06094ffaf891fdd34b95c40e197e58413466d14d36d2e6 | 2154839
+bls_g1_multiply 0xa061cdfcf9339ffd4e7996adcf12e1776882629dc084254f74faf150c718420e4b34940f173aca1c32d0dc6d6a776fb4 33 => 0xa64f6ed6aab62fdae581b3b658a1e0d39e891bd020cf4dcbb8894a690aadd6721de78b30080f525cf84c08815ea4eb07 | 2154839
+bls_g1_multiply 0xb66837657cd1564bb04e9acb1efd96b53ef90502bb000e6ca44da7e07f655e6fb50606f3cc47fd936ba4108ccf592625 39 => 0x8ebb948245a6a17c55ac85115ce0ee7c6296fb8569f02c6c75a2ee0b9812fe39ad76c34959df0e5f15b1d56a0b788d19 | 2154839
+bls_g1_multiply 0x8f3eb0208a3f0808df538ebc1f582d01c62baabe12842171f7d6c01f9b31f7c9d386190d9349d53378998d0162c7ecb2 93 => 0x887004d9a617003c6d62a29bfc25fa03299ca723db860d72ed502dbd6ce706df0c98f0365a7b6c8d36b446e65fca7707 | 2154839
+bls_g1_multiply 0x8b202593319bce41b090f3309986de59861ab1e2ff32aef871d83f9aac232c7253c01f1f649c6f69879c441286319de4 -22 => 0xa68e15cafe9d77c4ab64d0d5741efb33488ac097fd590db1cfc02e5972a601077651a1ad872e860535e8b6a74c8f564e | 2154839
+bls_g1_multiply 0xae9e0aed7144528e7629157798a88790d0941ecbf2d267eaf2f5abfc21a9438b415355afb7f76963def089ec1fd108a4 -74 => 0xb87fe1405ab0806df2283784d848aa554870045738c7c4fa38d51bb6ff284103f8e8f01b00f2eaa18984c1b317722733 | 2154839
+bls_g1_negate 0xb756ecad886b575d2c95014c20bcd9b52f1f72e3b5fbc657bacf1e72be707f9aaf76a9156a1628b99d873036beecec10 => 0x9756ecad886b575d2c95014c20bcd9b52f1f72e3b5fbc657bacf1e72be707f9aaf76a9156a1628b99d873036beecec10 | 471259
```

Comment:
This seems like it's way too expensive when you consider it's just copying the value and changing a bit. There's not much point in making it cost more than `logxor` with a constant.

Reply:
This operator also validates that the point is on the curve, which isn't trivial. I could implement this as just a bit operation, but it would make it accept invalid points, I believe.

Reply:
If we're concerned about invalid (ie. untrusted) points, we could have a separate operator to validate them. Or just do `(g1_add untrusted_point)`, which should also do an implicit validation. (Just spitballing here.)

Reply:
I think it's not likely that we're going to take a point and negate it and not perform any other operations on it that would validate it, so not validating here may not be a big concern.